File: app-java/src/main/java/com/java/bilibili/ui/main/fragment/impl/HomeImpl.java
package com.java.bilibili.ui.main.fragment.impl;

import com.java.bilibili.base.BasePresenter;
import com.java.bilibili.base.BaseView;
import com.java.bilibili.modle.Data;


public class HomeImpl {
    public interface View extends BaseView {
        void initDataInfo(Data data);
    }

    public interface Presenter extends BasePresenter<View> {
        void getAnimaData(int tid, int pn);
    }
}

File: app-java/src/main/java/com/java/bilibili/ui/main/fragment/impl/HomeImpl.kt
package com.kotlin.bilibili.ui.main.fragment.impl

import com.kotlin.bilibili.base.BasePresenter
import com.kotlin.bilibili.base.BaseView
import com.kotlin.bilibili.modle.Data

/**
 * Created by Eric on 2017/11/13 0013.
 */
interface HomeImpl {
    interface View : BaseView {
        fun initDataInfo(data:Data)
    }

    interface Presenter : BasePresenter<View> {
        fun getAnimaData(tid: Int,pn:Int)
    }
}